### DISCUSSION TITLE
单机多卡微调报错问题

### DISCUSSION THREAD

**Discussion by zainCSU** [zai-org/ChatGLM3#1142] [Q&A]
使用3090 24g双卡训练报错
![d119bab636a2dcc61284d5662cebfac](https://github.com/THUDM/ChatGLM3/assets/161547938/af9b463f-5ce1-4214-9de4-ae64b4f1cc04)
使用单卡显示显存不够
![6ff7401bb8e88a63d67f3cad1b47acd](https://github.com/THUDM/ChatGLM3/assets/161547938/85a0a152-c61a-4058-a334-5fc23db28ee1)
有没有办法解决这个报错或者降低训练的显存要求，batchsize已经写1了

**Comment by Duperr** [comment]
用的啥微调方法啊？把长度减半如何？

**Reply by zainCSU** [reply]
ptuning，是指把训练集的长度减半吗？
